kie-maven-plugin/IntegrationTests.md

```diff
@@ -14,6 +14,19 @@ Test compilations and reports may be found under _target/it/{module_name}_ direc
 _target/it/{module_name}/build.log_ will contain overall build output, while additionally test reports may be found under
 _target/it/{module_name}/target/failsafe-reports_ and _target/it/{module_name}/target/surefire_
 
+
+Skipping/selecting tests
+========================
+To completely disable execution of those tests, set the _invoker.skip_ parameter to false:
+
+    mvn clean install -Dinvoker.skip=true
```

Comment:
What about configure `maven-invoker-plugin` also to consider `skipTests` property like it has been done in Kogito ( see https://github.com/kiegroup/kogito-runtimes/blob/master/kogito-build-parent/pom.xml#L1234-L1235 )?
I think it simplify the usage for people that are not familiar with invoker plugin (and then of course add this information to this doc)

Reply:
@danielezonca 
Thanks for suggestion. Put together that with @mariofusco observation (and general standards) I've used the "skipITs" parameter.